client/battle/CBattleInterface.cpp

```diff
@@ -1810,14 +1846,16 @@ void CBattleInterface::endAction(const BattleAction* action)
 {
 	const CStack *stack = curInt->cb->battleGetStackByID(action->stackNumber);
 
-	if(action->actionType == Battle::HERO_SPELL)
+	if(action->actionType == EActionType::HERO_SPELL)
 		setHeroAnimation(action->side, 0);
 
-	if (stack && action->actionType == Battle::WALK &&
-		!creAnims[action->stackNumber]->isIdle()) //walk or walk & attack
-	{
-		pendingAnims.push_back(std::make_pair(new CMovementEndAnimation(this, stack, action->destinationTile), false));
-	}
+//???
+
+//	if (stack && action->actionType == EActionType::WALK &&
+//		!creAnims[action->stackNumber]->isIdle()) //walk or walk & attack
+//	{
+//		pendingAnims.push_back(std::make_pair(new CMovementEndAnimation(this, stack, action->destinationTile), false));
+//	}

```

Comment:
you can delete that comments part too, even if someone will need it, it will be faster to write again than understand for what it is